index.html

```diff
@@ -9298,6 +9298,279 @@
         </section>
       </div>
     </section>
+    <section id="rtcquicstream-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+        <pre class="idl">
+        [ Exposed=Window ]
+interface RTCQuicStream {
+    readonly        attribute RTCQuicTransport    transport;
+    readonly        attribute RTCQuicStreamState  state;
+    readonly        attribute unsigned long       bufferedAmount;
+    void                  close ();
+    void                  write (ArrayBuffer data);
+    ArrayBuffer           read ();
```

Comment:
For this PR, I recommend read() being "infinite buffer", in the sense that if you don't call read(), it will just buffer forever.  So you better call it!   And deal with better buffering and back pressure later.

Reply:
Have removed text relating to read buffering.